transaction-service/src/main/resources/application.properties

```diff
@@ -8,4 +8,6 @@ spring.jpa.generate-ddl=true
 spring.jpa.hibernate.ddl-auto=create
 spring.jpa.properties.hibernate.jdbc.lob.non_contextual_creation=true
 
-server.port=8500
\ No newline at end of file
+server.port=8500
+
+transaction.months=24
```

Comment:
transaction months means nothing to me
app.transactionService.monthsToExpire or something